tests/src/test/java/org/wso2/micro/gateway/tests/serviceDiscovery/EtcdSupportTestCase.java

```diff
@@ -189,20 +182,17 @@ public void testEtcdSupport() throws Exception {
 
         String[] args = {"-e", etcdUrlParameter, "-e", etcdUsernameParameter, "-e", etcdPasswordParameter, "-e",
                 pizzaShackProdParameter, "-e", pizzaShackSandParameter, "-e", etcdTimerParameter};
-        microGWServer.startMicroGwServer(balPath, args);
```

Comment:
Are disabling this test class? If not why we don't need to start mgw?

Reply:
We have started the server in the Before class method. Not starting it per test case